pypit/arparse.py

```diff
@@ -3779,10 +3779,14 @@ def load_sections(string):
         xyarrx = [0, 0]
     else:
         xyarrx = xyrng[0].split(':')
+        if len(xyarrx[0]) == 0: xyarrx[0] = 0
```

Comment:
Can you possible comment this code (a bit).  
Pretty hard to figure out what is going on (for me).